config/sync/core.entity_view_display.node.stanford_news.default.yml

```diff
@@ -16,6 +16,7 @@ dependencies:
     - field.field.node.stanford_news.su_news_hide_social
     - field.field.node.stanford_news.su_news_publishing_date
     - field.field.node.stanford_news.su_news_source
+    - field.field.node.stanford_news.su_news_tags
```

Comment:
Hi! We don't have any plans to display these on the News or Basic Pages at this time. Let's remove this! I should have specified that we just needed the field on the edit form and the taxonomies!

Reply:
I think you mentioned this. I thought I removed these but it looks like I dropped the ball. Sorry.